server/resp/src/test/java/org/infinispan/server/resp/SortedSetCommandsTest.java

```diff
@@ -244,15 +246,510 @@ public void testZPOPMAX() {
    public void testZSCORE() {
       assertThat(redis.zscore("not_existing", "no_existing")).isNull();
       redis.zadd("people", ZAddArgs.Builder.ch(),
-            ScoredValue.just(-10, "tristan"),
-            ScoredValue.just(1, "ryan"),
-            ScoredValue.just(17, "vittorio"),
-            ScoredValue.just(18.9, "fabio"),
-            ScoredValue.just(18.9, "jose"),
-            ScoredValue.just(18.9, "katia"),
-            ScoredValue.just(21.9, "marc"));
+            just(-10, "tristan"),
+            just(1, "ryan"),
+            just(17, "vittorio"),
+            just(18.9, "fabio"),
+            just(18.9, "jose"),
+            just(18.9, "katia"),
+            just(21.9, "marc"));
       assertThat(redis.zscore("people", "jose")).isEqualTo(18.9);
       assertThat(redis.zscore("people", "takolo")).isNull();
       assertWrongType(() -> redis.set("another", "tristan"), () ->  redis.zscore("another", "tristan"));
    }
+
+   public void testZRANGE() {
```

Comment:
More of a nitpick. We really cover each one separately with multiple combinations. Can we add just a simple one that mixes the commands?

Reply:
methods are already too big already. lettuce does not support all the options in zrange, but keeps the old commands. the code is the same, with the same parameters sent.